infrastructure/helm/quantumserverless/charts/gateway/values.yaml

```diff
@@ -52,10 +52,10 @@ serviceAccount:
 
 podAnnotations: {}
 
-podSecurityContext: {}
-#  fsGroup: 1001
-#  runAsUser: 1001
-#  runAsGroup: 1001
+podSecurityContext:
```

Comment:
Did we need this change for COS?

Reply:
Agree, for COS probably not.

Reply:
Solved in [e2029a9](https://github.com/Qiskit-Extensions/quantum-serverless/pull/668/commits/e2029a9d291cd2e871a2356cb4ca838252e177b4)